src/meshapi/admin.py

```diff
@@ -148,7 +199,9 @@ class MemberAdmin(admin.ModelAdmin):
     search_fields = [
         # Search by name
         "name__icontains",
-        "email_address__icontains",
+        "primary_email_address__icontains",
+        "stripe_email_address__icontains",
+        "secondary_email_addresses__icontains",
```

Comment:
Ooops

Reply:
I hope you're pleased with yourself >:(

Reply:
And I can't use the `all_email_addresses` property to filter because the searches happen at the DB level.

https://stackoverflow.com/a/1205389